.github/workflows/workflow.yml

```diff
@@ -356,6 +356,9 @@ jobs:
       name: Unit tests
       run: |
         tox -e py${{ matrix.python_version }} -- -m 'not integration and not unstable'
+    - name: Plugin tests
+      run: |
+        tox -e plugins-py${{ matrix.python_version }} -- --cov-append -m 'not integration and not unstable'
```

Comment:
note `--cov-append`. An alternative is to generate different coverage file for each plugin and then merge with `coverage combine`:

https://coverage.readthedocs.io/en/latest/cmd.html#cmd-combine